syntax = "proto3";

package helloworld;

service Hello {
	rpc SayHello (HelloRequest) returns (HelloReply) {}
}

message HelloRequest {
	string name = 1;
  string age = 2;
}

message HelloReply {
	string message = 1;
  string age = 2;
}